install.py

```diff
@@ -30,28 +46,68 @@ def ensure_init_files(workspace: str):
                 with open(init_path, 'w') as f:
                     f.write("")
 
-def install_custom_node_req(workspace: str):
-    custom_nodes_path = os.path.join(workspace, "custom_nodes")
-    for folder in os.listdir(custom_nodes_path):
-        folder_path = os.path.join(custom_nodes_path, folder)
-        req_file = os.path.join(folder_path, "requirements.txt")
 
-        if os.path.isdir(folder_path) and os.path.isfile(req_file):
-            logger.info(f"Installing requirements for {folder}...")
-            subprocess.run(["pip", "install", "-r", req_file], check=True)
+def download_and_extract_ui_files(version: str):
+    """Download and extract UI files to the workspace"""
 
+    output_dir = os.path.join(os.getcwd(), "nodes", "web")
+    pathlib.Path(output_dir).mkdir(parents=True, exist_ok=True)
+    base_url = urllib.parse.urljoin("https://github.com/yondonfu/comfystream/releases/download/", f"v{version}/static.tar.gz")
+    
+    # Create a temporary directory instead of a temporary file
+    with tempfile.TemporaryDirectory() as temp_dir:
+        # Define the path for the downloaded file
+        download_path = os.path.join(temp_dir, "static.tar.gz")
+        
+        # Download tar.gz file
+        logger.info(f"Downloading {base_url}")
+        try:
+            urllib.request.urlretrieve(base_url, download_path)
+            
+            # Extract contents
+            logger.info(f"Extracting files to {output_dir}")
+            with tarfile.open(download_path, 'r:gz') as tar:
+                tar.extractall(path=output_dir)
+        except Exception as e:
+            logger.error(f"Error downloading or extracting files: {e}")
+            raise
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Install custom node requirements")
     parser.add_argument(
-        "--workspace", default=None, required=True, help="Set Comfy workspace"
+        "--workspace", default=None, required=False, help="Set Comfy workspace"
     )
     args = parser.parse_args()
-
-    logger.info("Installing custom node requirements...")
-    install_custom_node_req(args.workspace)
     
-    logger.info("Ensuring __init__.py files exist in ComfyUI directories...")
-    ensure_init_files(args.workspace)
+    workspace = args.workspace
+    if workspace is None:
```

Comment:
@eliteprox don't we have an environmental variable? If not no problem.

Reply:
Added that env variable here https://github.com/yondonfu/comfystream/blob/93164d1830c627408059149f95be7138c887cede/install.py#L78